scss/_utilities/_variables.scss

```diff
@@ -74,18 +74,22 @@ $weight-sbold: 600;
 $weight-bold: 700;
 
 // Typography
-$font-hero: 54px;
-$font-larger: 36px;
-$font-large: 32px;
-$font-medium: 24px;
-$font-regular: 18px; // base font size
-$font-small: 16px;
-$font-smaller: 13px;
+$font-gigantic: 83.989px;
```

Comment:
It's a scale based on `18px`, then multiplied or divided by `1.250`. so like 18*1.250 = 22.500.

Here's the full table

Pixel Value  | Ems  | Heading level
-------|-------|-------
|11.520px | 0.640em || 
|14.400px | 0.800em | Captions |
|18.000px | 1.000em | Body/Gamma |
|22.500px | 1.250em | Delta |
|28.125px | 1.563em | Beta |
|35.156px | 1.953em ||
|43.945px | 2.441em ||
|53.750px | 2.986em | Alpha |
|67.187px | 3.722em ||
|83.983px | 4.666em | Super big |